clean up micromamba instructions

christian-rauch opened the pull request:
The "Getting Started" documentation uses a confusing combination of `conda`, `mamba` and `micromamba` commands (https://github.com/RoboStack/robostack.github.io/issues/108).  Fix this by changing everything to using `micromamba` only.

Tobias-Fischer commented:
Agreed - can we just have a third tab with conda instructions, and rename the mamba tab to micromamba?

Tobias-Fischer commented:
Thanks a lot by the way @christian-rauch for your contribution, highly appreciated :)!

Tobias-Fischer commented:
@traversaro - are you happy to merge?

traversaro commented:
> @traversaro - are you happy to merge?

Sorry, I had missed the changes! Yes the changes are perfectly fine, I have just some doubt of suggesting to install miniconda, see https://github.com/RoboStack/robostack.github.io/pull/111/files#r2348238184 . If we can stick to suggest miniforge everything is fine for me.

traversaro commented on docs/GettingStarted.md:
````diff
@@ -3,121 +3,131 @@
-            ```
+=== "Conda"
+    ## Install conda
+    On Debian- and RPM-based distributions, we recommend the [Debian and RPM repository for miniconda](https://docs.conda.io/projects/conda/en/stable/user-guide/install/rpm-debian.html). To source the base environment:
+    ```bash
+    source /opt/conda/etc/profile.d/conda.sh
````
> I don't remember seeing the ToS presented to me during the installation. I guess this is different from Anaconda, which contains additional packages? On https://www.anaconda.com/legal/terms/terms-of-service#UseofSpecificOfferings:

Unfortunately, as far as I know `miniconda` is subject to the ToS exactly like Anaconda full distribution, see https://www.anaconda.com/docs/getting-started/miniconda/main#is-miniconda-free-for-me .

traversaro replied:
A common trick used by IT departments to avoid violation is to block anaconda.com and allow anaconda.org (that hosts conda-forge). Those Debian repos are on anaconda.com, so I would prefer to avoid suggesting their use.

traversaro replied:
Documenting *also* `conda config --env --remove channels defaults` is perfectly fine for me, but given my past problems with ToS I strongly prefer not to document at all any installation method that may eventually install the `defaults` channel, as I find it confusing (for most users miniconda == defaults and miniforge == conda-forge, so I think it could be confusing to suggest miniconda) and as if a users does not follow all steps, it may stil end up with a conda installation that by default install packages from the `defaults` channel.

To take the issue from another point of view, that is the problem of just linking https://github.com/conda-forge/miniforge or https://conda-forge.org/download/ and call it a day? If you are worried about the automatic base activation (that is indeed annoying) we can document to disable it via `conda config --set auto_activate_base false`.

christian-rauch commented:
I switched the recommendation for conda back to Miniforge and also added a warning about the `defaults` channel with more information, as it previously was not clear why the `defaults` channel was removed. I kept the DEB/RPM repo for Miniconda, as this is IMHO the cleanest way to install and remove conda again. If Miniforge is provided as DEB/RPM for clean installation and removal, I would recommend this in place of Miniconda.

traversaro reviewed (approved):
Thanks, as long as miniforge is clearly the recommendation, I am ok with this.

Tobias-Fischer commented:
Thanks a lot @christian-rauch
